.github/workflows/release-typegen.yaml

```diff
@@ -0,0 +1,35 @@
+name: release-typegen
```

Comment:
Why pull this into its own workflow?

Reply:
typgen binary and the CLI have different release cycles, so it doesn't make sense to couple them.